spec: Rename ABCI++ in the spec (#2712)
cometbft/cometbft: spec/abci/abci++_comet_expected_behavior.md

> ## Adapting existing Applications that use ABCI

Reviewer: legacy ABCI? previous ABCI versions?

Author: changed to legacy ABCI

> In some cases, an existing Application using the legacy ABCI may need to be adapted to work with ABCI

Reviewer: ```suggestion
In some cases, an existing Application using the legacy ABCI may need to be adapted to work with new version of ABCI
```

> with as minimal changes as possible. In this case, of course, ABCI will not provide any advantage with respect

Reviewer: ```suggestion
with as minimal changes as possible. In this case, of course, new ABCI versions will not provide any advantage with respect
```

> to the existing implementation, but will keep the same guarantees already provided by ABCI.

Reviewer: ```suggestion
to the legacy ABCI implementation, but will keep the same guarantees.
```
